src/Accelerators/Accelerator.cpp

```diff
@@ -25,5 +25,25 @@ const llvm::SmallVectorImpl<Accelerator *> &Accelerator::getAccelerators() {
   return acceleratorTargets;
 }
 
+// Help to print accelerator kinds.
+static std::map<Accelerator::Kind, std::string> mapKind2Strings;
+
+std::ostream &operator<<(std::ostream &out, const Accelerator::Kind kind) {
+  if (mapKind2Strings.size() == 0) {
+    APPLY_TO_ACCELERATORS(ACCEL_CL_ENUM_TO_STRING, mapKind2Strings);
+    mapKind2Strings[Accelerator::Kind::NONE] = "NONE";
+  }
+  return out << mapKind2Strings[kind];
+}
+
+llvm::raw_ostream &operator<<(
+    llvm::raw_ostream &out, const Accelerator::Kind kind) {
+  if (mapKind2Strings.size() == 0) {
```

Comment:
[Suggestion]: if (mapKind2Strings.empty()) {

Reply:
PS: I am usually pretty good at applying suggestions to other places, I appreciate your thoroughness.